### PR TITLE
CI: upgrade to llvm 9.0.0

### PR DESCRIPTION
**Pull request by pchickey**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -34,17 +34,17 @@ jobs:
...
         echo "::add-path::$CLANG_DIR"
         echo "::set-env name=WASM_CC::$CLANG_DIR/clang"
       if: matrix.os == 'macos-latest'
 
     - name: Install clang (Linux)
       shell: bash
       run: |
-        curl -sSf http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz | tar xJf -
-        export CLANG_DIR=`pwd`/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-16.04/bin
+        curl -sSf https://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz | tar xJf -
```
Comment (sbc100) [inline .github/workflows/main.yml]: http!  nice catch.

**Comment by sbc100** [comment]
Do we want the next version of the SDK be based on llvm 9.0?

It would mean that anyone who wants to bring thier own llvm or use system llvm also need to be on version 9?   I think that requirement is fine, given that its the current stable version.  What do you think @sunfishcode ?

**Comment by pchickey** [comment]
We've historically tracked the latest stable, so I think its a good idea.

**Comment by sunfishcode** [comment]
Tracking stable makes sense to me.

@pchickey How hard would it be to add CI rules that test that the SDK still builds using LLVM 8.0? As far as I'm aware, that should still work; we just don't have any tests for it right now.

File: .github/workflows/main.yml (hunk 1 above)
Comment (pchickey) [reply .github/workflows/main.yml]: the link URL from the llvm release website is http, not https. Someone over there should please update that!

**Comment by pchickey** [comment]
I could add a build matrix dimension for llvm version, we'd have to duplicate some steps but it probably wouldn't be too big of a deal.

**Comment by sbc100** [comment]
I'm not sure its worth the effort TBH.  I think its ok to say "this is the version we support and test against, other versions YMMV".

**Comment by sbc100** [comment]
Should we cut a release of the SDK based on llvm 9.0.0 to keep it all in sync?

**Comment by pchickey** [comment]
Its about time, yeah. @sunfishcode how much of your current stuff do you want to land before an SDK release?

**Pull request merged**